msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,168 @@ The following image shows the default view of the Adaptive Card when you have no
+
+  The following examples explain how the elements can be made visible or hidden based on the condition with `targetWidth` and `hostWidth`: 
+
+  * If you set `targetWidth` to `standard`, the element is only visible when the `hostWidth` property is set to `wide`.
+
+  * If you set `targetWidth` to `atMost:narrow`, the element is only visible if the `hostWidth` property is set to `veryNarrow` or `narrow`.
+
+  * If you set `targetWidth` to `atLeast:standard`, the element is only visible if the `hostWidth` property is set to `standard` or `wide`.
+
+#### Sample Adaptive Card responsive layout
+
+To make a responsive Adaptive Card, your app must include the elements from the following code sample:
+
+``` json
+{ 
+  "type": "AdaptiveCard", 
+  "body": [ 
+    { 
+      "type": "ColumnSet", 
+      "columns": [ 
+        { 
+          "type": "Column", 
+          "items": [ 
+            { 
+              "type": "Image", 
+              "style": "Person", 
+              "url": "https://pbs.twimg.com/profile_images/3647943215/d7f12830b3c17a5a9e4afcc370e3a37e_400x400.jpeg", 
+              "size": "Small"
+            } 
+          ], 
+          "width": "auto" 
+        }, 
+        { 
+          "type": "Column", 
+          "spacing": "medium", 
+          "items": [ 
+            { 
+              "type": "TextBlock", 
+              "weight": "Bolder", 
+              "text": "Matt Hidinger", 
+              "wrap": true 
+            }, 
+            { 
+              "type": "TextBlock", 
+              "targetWidth": "narrow", 
+              "spacing": "None", 
+              "text": "Principal Program Manager", 
+              "isSubtle": true, 
+              "wrap": true 
+            } 
+          ], 
+          "width": "auto", 
+          "verticalContentAlignment": "center" 
+        }, 
+        { 
+          "type": "Column", 
+          "targetWidth": "atLeast:standard", 
+          "spacing": "medium", 
+          "items": [ 
+            { 
+              "type": "TextBlock", 
+              "text": "Principal Program Manager", 
+              "isSubtle": true, 
+              "wrap": true 
+            } 
+          ], 
+          "width": "stretch", 
+          "verticalContentAlignment": "center" 
+        } 
+      ] 
+    } 
+  ], 
+  "$schema": "https://adaptivecards.io/schemas/adaptive-card.json", 
+  "version": "1.5" 
+} 
+```
+
+#### Responsive Adaptive Card layout design guidelines
+
+##### `targetWidth` set to `wide`
+
+You can use the following design guidelines for the `targetWidth` property set to `wide`:
+
+:::image type="content" source="../../assets/images/Cards/design-layout-wide.png" alt-text="Screenshot shows the design guidelines for a responsive adaptive card when you set target width to wide.":::
```

Comment:
The various screenshots you added here don't seem to be useful for developers. I wouldn't know what to do with them.

If we're going to provide recommendations, we should take a non-responsive card as an example and show how it doesn't look good at certain widths. Then show how to update it using `targetWidth` to make it responsive.